README.md

```diff
@@ -1 +1,38 @@
 # Pipeline Run Prometheus Counter
+
+Expose counter metrics for various pipeline runs for the Prometheus text format.
+
+This is useful if you have a pipeline such as Tekton or Jenkins and you need
+to keep track of success and/or failure rates of Pipeline runs in Prometheus.
+
+These pipeline runs are stored in a simple SQLite database.
```

Comment:
The idea is you store the SQLite database on a persistent volume. It could be any database but it can't be stored in memory or a text file since the pod will have multiple WSGI workers to be able to respond to both POST and GET requests at the same time.

Reply:
Is it safe to use SQLite from a persistent volume mount and access it using multiple processes?

Reply:
Yes, but if there are multiple writers, the database will be locked until the write is completed.

Reply:
I'm just curious if there are any requirements for the volume mount that might be missing.

Anyway, writers will be very rare. Write would occur maybe every ~15minutes; read could happen every few seconds.